content/articles/introduction-to-automation-pyautogui/index.md

```diff
@@ -0,0 +1,141 @@
+
+In this article, we are going to take a look at a simple automation task that involves opening Google Chrome, clicking on the address search bar, typing the URL to YouTube, searching for a video(in this case, a snap), clicking on it to play and finally minimizing the Google Chrome window. 
+
+
+
+#### Prerequisites
+To follow along in this article, you need to:
+- Install Python on your machine. To download Python, click [here](https://www.python.org/downloads/).
+- Have a code editor, preferably VS code.
+- Have a browser installed in your machine, preferably Google Chrome.
+
+#### Goal
+By the end of this article, you should be able to:
+- Automate mouse movements and clicks.
+- Automate keyboard key presses.
+- Automate typing into a textbox.
+
+#### Setting up our Project
+To get started, create a folder named **Automating Basic Tasks**. Open the folder and create a file named `automation.py` We are using the `.py` file extension because it's a python file. Open the file with your code editor. In this case, we are using VS Code.
+
+Launch the VS Code integrated terminal and type in the following command:
+
+```bash
+py -m pip install pyautogui
+```
+
+The above command is used to install PyAutoGUI on windows. To install for a different OS, click [here](https://pyautogui.readthedocs.io/en/latest/install.html) and find the instructions.
+
+> **Disclaimer:** In this article, I have used coordinates that correspond to my screen size and the positions of the specific points where items I want to perform the automation operations are located. To follow along with this article, you need to use the MouseInfo Python library to get the correct coordinates that
+correspond to the position that you want to perform any of these automation operations. 
+
+> Adjust the duration of the time it takes to perform an action that suits your internet speed. Using short duration when on a slow internet connection may lead to clicking of a the unintended point as the page will not have fully loaded.
+
+#### Getting the Coordinates
+PyAutoGUI uses the screen coordinates to get the exact position to perform various actions like clicking, typing, scrolling, pressing, and dragging.
+
+MouseInfo is a module that comes with the PyAutoGUI library. You need to write and run the code below in a separate file in that,
+do not include it in the code that we will use during the automation task. 
+
+```python
+import pyautogui
+
+pyautogui.mouseInfo()
+```
+
+**Code explanation:**
+- `import pyautogui` - This line of code is used to import the PyAutoGUI library into our project.
+
+- `pyautogui.mouseInfo()` - We use this command to get the x,y coordinates of a point on the screen.
+
+Run the above code. It will display a dialog box that displays the point's x and y coordinates where the cursor is positioned. If you want to get the coordinates of another point on the screen, move the cursor to that point, and it will give you the coordinates.
```

Comment:
What Is the convention for the origin? Is it bottom left? Can you elaborate on that.